### PR TITLE
feat: set history data to `SeverityStatusByService` widget

### PR DESCRIPTION
**Pull request by yuda110**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [ ] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [x] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description
스크린샷 2023-06-13 오후 9 36 51

### CODE REVIEW

File: apps/web/src/services/dashboards/widgets/asset-widgets/severity-status-by-service/SeverityStatusByServiceWidget.vue [hunk 1]
```diff
@@ -137,19 +142,34 @@ const fetchData = async (): Promise<Data[]> => {
...
+        refinedData.push({
+            service: service as string,
             severity,
-            complianceStatus,
-            value: complianceStatus === 'FAIL' ? result.fail_finding_count : result.pass_finding_count,
-        };
+            value,
+        });
     });
+    return refinedData;
```
Comment (yuda110) [inline apps/web/src/services/dashboards/widgets/asset-widgets/severity-status-by-service/SeverityStatusByServiceWidget.vue:172]: 🥲

File: apps/web/src/services/dashboards/widgets/asset-widgets/severity-status-by-service/SeverityStatusByServiceWidget.vue [hunk 2]
```diff
@@ -95,40 +102,38 @@ const fetchData = async (): Promise<Data[]> => {
         const apiQueryHelper = new ApiQueryHelper();
         apiQueryHelper
             .setFilters(state.consoleFilters)
-            .addFilter({ k: 'ref_cloud_service_type.labels', v: 'Compliance', o: '=' });
-        const results = await SpaceConnector.clientV2.inventory.cloudService.analyze({
+            .addFilter({ k: 'ref_cloud_service_type.labels', v: 'Compliance', o: '=' })
+            .addFilter({ k: 'key', v: ['fail_finding_count', 'pass_finding_count'], o: '' });
+        const prevMonth = dayjs.utc(state.settings?.date_range?.end).subtract(1, 'month').format(DATE_FORMAT);
+        const { results } = await SpaceConnector.clientV2.inventory.cloudServiceStats.analyze({
```
Comment (yuda110) [inline apps/web/src/services/dashboards/widgets/asset-widgets/severity-status-by-service/SeverityStatusByServiceWidget.vue:108]: description: `inventory/cloud-service/analyze` -> `inventory/cloud-service-stats/analyze`

File: apps/web/src/services/dashboards/widgets/asset-widgets/severity-status-by-service/SeverityStatusByServiceWidget.vue (hunk 1 above)
Comment (yuda110) [reply apps/web/src/services/dashboards/widgets/asset-widgets/severity-status-by-service/SeverityStatusByServiceWidget.vue:172]:
```json
{
    "results": [
        {
            "value": 0.0,
            "severity": [
                "MEDIUM"
            ],
            "unit": "Count",
            "status": [
                "PASS"
            ],
            "service": "account",
            "date": "2023-06",
            "key": "fail_finding_count"
        },
        {
            "status": [
                "PASS"
            ],
            "key": "pass_finding_count",
            "service": "account",
            "value": 9.0,
            "severity": [
                "MEDIUM"
            ],
            "unit": "Count",
            "date": "2023-06"
        }
    ],
    "more": false
}
```

File: apps/web/src/services/dashboards/widgets/asset-widgets/severity-status-by-service/SeverityStatusByServiceWidget.vue [hunk 3]
```diff
@@ -137,19 +142,34 @@ const fetchData = async (): Promise<Data[]> => {
...
+const refineData = (data?: Data[]): RefinedData[] => {
+    if (!data?.length) return [];
+    const serviceSet = new Set();
+    data.forEach((result) => serviceSet.add(result.service));
+    const refinedData: RefinedData[] = [];
+    serviceSet.forEach((service) => {
+        const targetServiceDataList = data.filter((result) => result.service === service);
+        const statusList = flattenDeep(targetServiceDataList.map((d) => d.status));
+        const status = statusList.includes('FAIL') ? 'FAIL' : 'PASS';
+        let severity: Severity = 'PASS';
```
Comment (WANZARGEN) [inline apps/web/src/services/dashboards/widgets/asset-widgets/severity-status-by-service/SeverityStatusByServiceWidget.vue:154]: nitpick: please separate logics to another function for the loosely related concern. e.g. `getStatus()`

**Review by WANZARGEN — APPROVED** [review]
I totally understood the requirements and the reason why this change was required thanx to your explanation!
I left a very small nitpick comment.

**Review by sulmoJ — APPROVED** [review]
LGTM!
I only checked code convention!

**Pull request merged**